### PR TITLE
Unify interfaces

### PR DESCRIPTION
**Pull request by MksmOrlov**
Перед мержем обновить документацию

### CODE REVIEW

**Comment by MksmOrlov** [comment]
> Fails on EmptyInputStructure test. Maybe change argument description order in documentation according to their rrel number.

It was failing before unifying too. This PR doesn't touch EmptyInputStructure test logic. There is a TODO mark to fix it. 
Arguments order is correct in documentation and code

File: docs/scl-machine.tex [hunk 1]
```diff
@@ -162,29 +125,28 @@
...
+    \scnfileitem{Входная структура интерпретируется как параметры, которые подставляются в логические формулы, а не как структура, в которой нужно искать.}
 \end{scnrelfromset}
 
 \begin{scnrelfromset}{преимущества текущего состояния}
     \scnfileitem{Проверка передаваемых параметров}
     \begin{scnindent}
         \scntext{примечание}{Если не передано множество правил, успешность завершения агента зависит от того, была ли достигнута цель.}
         \scntext{примечание}{Если не передана входная структура, используется База знаний IMS.}
-        \scntext{примечание}{Если не передана выходная структура, результат генерации никуда не записывается.}
+        \scntext{примечание}{Если не передана выходная структура, создаётся новыая, в которую записывается результат генерации.}
```
Comment (AlexandrZagorskiy) [inline docs/scl-machine.tex:148]:
```suggestion
        \scntext{примечание}{Если не передана выходная структура, создаётся новая, в которую записывается результат генерации.}
```

**Review by SHtress — APPROVED** [review]

File: docs/scl-machine.tex [hunk 2]
```diff
@@ -162,29 +125,28 @@
...
         \scntext{примечание}{Если не передан шаблон цели, агент проверяет выполнимость каждого правила на входной структуре.}
     \end{scnindent}
-    \scnfileitem{Сохранён интерфейс предыдущей реализации агента с тремя  аргументами.}
+    \scnfileitem{При передачи параметров агенту по предыдущему варианту реализации с тремя аргументами, агент работает корректно.}
```
Comment (AlexandrZagorskiy) [inline docs/scl-machine.tex:151]:
```suggestion
    \scnfileitem{Агент работает корректно при передаче параметров в соответствии с предыдущим вариантом его реализации.}
```

File: scl/cxx/inferenceModule/test/units/TestManagerModule.cpp [hunk 3]
```diff
@@ -29,189 +30,77 @@ void initialize()
...
-  ScAddr argumentSet = context.HelperResolveSystemIdtf(MESSAGE_SINGLETON);
-  EXPECT_TRUE(argumentSet.IsValid());
-
-  DirectInferenceManager inferenceManager(&context);
-  ScAddr answer = inferenceManager.applyInference(targetTemplate, ruleSet, argumentSet);
-
-  EXPECT_TRUE(answer.IsValid());
-  EXPECT_TRUE(
-      context.HelperCheckEdge(InferenceKeynodes::concept_success_solution, answer, ScType::EdgeAccessConstNegPerm));
-}
```
Comment (AlexandrZagorskiy) [inline scl/cxx/inferenceModule/test/units/TestManagerModule.cpp:216]: why this tests were just removed? Or they are were moved to another file?

**Review by AlexandrZagorskiy — CHANGES_REQUESTED** [review]

File: scl/cxx/inferenceModule/test/units/TestManagerModule.cpp (hunk 3 above)
Comment (MksmOrlov) [reply scl/cxx/inferenceModule/test/units/TestManagerModule.cpp:216]: TestLogicModule handle it.
Also this tests were incorrect because of test structures

**Review by AlexandrZagorskiy — APPROVED** [review]

**Pull request merged**